src/libraries/System.Formats.Nrbf/tests/ArraySinglePrimitiveRecordTests.cs

```diff
@@ -20,10 +20,15 @@ public NonSeekableStream(byte[] buffer) : base(buffer) { }
 
     public static IEnumerable<object[]> GetCanReadArrayOfAnySizeArgs()
     {
-        foreach (int size in new[] { 1, 127, 128, 512_001, 512_001 })
+        foreach (int size in new[] { 1, 127, 128, 512_001 })
```

Comment:
Is `512_001` particularly interesting value to test for some reason? 

Would it be simpler to just change `512_001` to something smaller, like `20_001`?

Reply:
Nothing in particular, I am going to apply your suggestion.